schema/lc.proto

```diff
@@ -204,6 +221,9 @@ service LcService {
 	rpc ZScanExt (immudb.schema.ZScanRequest) returns (ZItemExtList){};
 	rpc HistoryExt(immudb.schema.HistoryRequest) returns (ItemExtList){};
 	rpc Feats(google.protobuf.Empty) returns (Features){};
+	rpc LabelsAdd(LabelsSetRequest) returns (LabelsSetResponse){};
```

Comment:
@byo / @qrdl / @arturmelanchyk , cosmetic, but as this is VCN api, mayby should use VCN prefix in method names, VCNLabelAdd, VCNLabelsRemove etc. Same like VCNSetArtifacts etc.

Reply:
also, as discussed today with @tomekkolo @qrdl @zaza81 , lets maybe rename these to `Uri`, or otherwise rename flags in VCN from `--uri-add` to `--label-add`?

Reply:
yes, rename this to URI